packages/firestore/src/api/database.ts

```diff
@@ -772,17 +772,20 @@ export class Transaction implements firestore.Transaction {
   }
 
   update(
-    documentRef: firestore.DocumentReference<unknown>,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    documentRef: firestore.DocumentReference<any>,
     value: firestore.UpdateData
   ): Transaction;
   update(
-    documentRef: firestore.DocumentReference<unknown>,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    documentRef: firestore.DocumentReference<any>,
     field: string | ExternalFieldPath,
     value: unknown,
     ...moreFieldsAndValues: unknown[]
   ): Transaction;
   update(
-    documentRef: firestore.DocumentReference<unknown>,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    documentRef: firestore.DocumentReference<any>,
```

Comment:
Since this "catch all" implementation is not part of the Public API, you could still use "unknown" here (if the TypeScript compiler allows it.

Optional, but it does provide a bit more type safety.